byzcoin/service.go

```diff
@@ -34,6 +34,9 @@ import (
 	uuid "gopkg.in/satori/go.uuid.v1"
 )
 
+// ContractsFn ...
```

Comment:
Finish this comment please. In particular mention: `Access to this map is not threadsafe, so it must only be written to from a package's init() function.`

The Go runtime is single-threaded while running all of the init() functions.